### PR TITLE
added hardhat project for pixel avatars

### PR DESCRIPTION
**Pull request by etherinsight**
This is a working version of the hardhat solidity project for pixel avatars.

### CODE REVIEW

**Comment by Dhaiwat10** [comment]
Else, this looks good to me. Let's get this merged and add tests?

**Review by Dhaiwat10 — APPROVED** [review]

File: pixel-avatar-hardhat/contracts/pxl-avatars.sol [hunk 1]
```diff
@@ -0,0 +1,1803 @@
...
+    function setBaseURI(string memory newBaseURI) public onlyOwner {
+        baseURI = newBaseURI;
+    }
+
+  
+
+    //Private sale minting (reserved for DevDao owners)
+    function mintWithDevDaoToken(uint256 DevDaoTokenId)
+        public
+        payable
```
Comment (Dhaiwat10) [inline pixel-avatar-hardhat/contracts/pxl-avatars.sol:1775]: Why is this payable tho?

Comment (ravsau) [reply pixel-avatar-hardhat/contracts/pxl-avatars.sol:1775]: good catch. Not sure why this was kept payable on my reference contract. I'll remove this.

Comment (etherinsight) [reply pixel-avatar-hardhat/contracts/pxl-avatars.sol:1775]: Reposting my comment as etherinsight again 😅 

good catch. Not sure why this was kept payable on my reference contract. I'll remove this.

**Pull request merged**